CHANGELOG.md

```diff
@@ -4,6 +4,6 @@ All notable changes to SDCcc will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-###
-- initial publicly available release
+## [5.0.0] - 2022-10-26
```

Comment:
```suggestion
## [Unreleased]

## [5.0.0] - 2022-10-26
```

Reply:
No, then unreleased would appear as the first entry in the release. Changelog has to be updated afterwards again.